addons/food/functions/fnc_hasCamelbak.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+	File Name: fnc_hasCamelbak.sqf
+	Author: Jak Keen
+*/
+
+#include "../script_component.hpp"
+
+private ["_item", "_return"];
+
+_item = "";
+_return = false;
+
+if(GVAR(camelbak_item) in (items player)) exitwith {
+    _item = _x;
```

Comment:
where does the ? `_x` come from?

Reply:
Seeing that this is called from https://github.com/acemod/ACE3/pull/6339/files#diff-ca5c2e394d87bed18d3bae861f257a91R23 which is not in a forEach loop. This is a script error.

Reply:
This is an error and has now been rewritten